src/components/UI/LiveHelpButton/LiveHelpButton.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import LiveHelpIcon from '@material-ui/icons/LiveHelp';
+
+import IconButton from '../IconButton';
+import messages from '../../Settings/Settings.messages';
+
+const propTypes = {
+  /**
+   * The component used for the root node. Either a string to use a DOM element or a component.
+   */
+  component: PropTypes.object,
```

Comment:
Should be: [`PropTypes.node`](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes)

Reply:
Corrected!